content/articles/2018-02-stringr-1-3-0.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "stringr 1.3.0"
+slug: stringr-1-3-0
+description: >
+    stringr 1.3.0 is now on CRAN.
+date: 2018-02-21
+photo:
+  url: https://unsplash.com/photos/dLTpk6N31Fc
+  author: Igor Ovsyannykov
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library(stringr)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are happy to announce that [stringr](http://stringr.tidyverse.org/) 1.3.0
+is now on CRAN. stringr provides a cohesive set of functions designed to make
+working with strings as easy as possible. For a complete list of changes, please
+see the [release notes](https://github.com/tidyverse/stringr/releases/tag/v1.3.0).
+
+## What's new?
+
+*  Two new wrappers from the [glue](http://glue.tidyverse.org/)
```

Comment:
"If you haven't heard of glue, you're in for a treat! Glue lets you easily interpolate ... into ..."

And show an example.

"Since stringr is loaded with tidyverse, this means that you can now access glues functionality without loading another package"